test/unit/services/azurerediscache/provision-spec.js

```diff
@@ -60,6 +60,27 @@ describe('RedisCache - Provision - PreConditions', function() {
     });
 });
 
+describe('RedisCache - Provision - PreConditions incorrect', function() {
+    var validParams = {};
+    var cp;
+        
+    before(function() {
+        validParams = {  /* missing parameters!! */
+            instance_id : 'b259c5e0-7442-46bc-970c-9912613077dd',            
+            azure : azure,
+            provisioning_result: '{\"provisioningState\":\"Creating\"}'
```

Comment:
You're correct - provisioningState shouldn't be part of the validParms for provision command.  In this particular case, I'm testing for missing parameters, so the location and everything else that is part of parameters object is missing intentionally.